Yoshi/Yoshi/Utility/UIWindowExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIWindowExtension.swift
+//  Yoshi
+//
+//  Created by Quentin Ribierre on 10/24/16.
+//  Copyright © 2016 Prolific Interactive. All rights reserved.
+//
+
+import UIKit
+
+
+// MARK: - UIWindow Extension.
+public extension UIWindow {
```

Comment:
Not sure about including this in Yoshi itself. Would this cause problems if I am trying to extend `UIWindow` in my project that imports Yoshi?